### ISSUE TITLE
Properties of Reference Implementations

### ISSUE THREAD

**Issue by ddutta** [mlcommons/training_policies#44]
What should constitute a reference implementation?

Some early thoughts: 

- GitHub issue with a link to a working doc 
-- algorithm details: e.g. network structure 
-- normalization 
-- pre-processing
-- hyper-param tuning
-- optimizations
-- GPU/accelerator assumptions and optimizations
-- datasets for validation: a small dataset for QA and some reference ones. 
-- reproducing the results 
- code should checked against this issue 
- checking and review guidelines 
-- get initial design doc reviewed 
-- check in test harness/infra with scripts
-- check in the implementation with a [WIP] tag, get initial reviews
-- at least 2 core +ve and no -ve reviews. Core reviewers are expected to run against small dataset at a minimum to OK. Ideally one of the reviews should be from a subject matter expert. Also reviewers should check for adherence to coding guidelines.

**Comment by petermattson** [comment]
Part of the this will be handled by 0.5 compliance list. But this is a bigger issue than submitters should handle, so putting in my inbox to do something at community level.

**Comment by petermattson** [comment]
Moved to the training code repo.
